### DISCUSSION TITLE
Little bug in Admin UI (date field not cleared)

### DISCUSSION THREAD

**Discussion by pgswow** [pocketbase/pocketbase#1730] [Q&A] [answered]
Hello! when clearing the date by clicking on the cross, the date field is not cleared, although when you save it later, the field becomes empty. PB v 0.12.1

**Comment by ganigeorgiev** [comment, marked as answer]
Hm, this seems to be a regression in [svelte-flatpickr](https://github.com/jacobmischka/svelte-flatpickr). Downgrading to v3.2.4 works as expected. I'll investigate it sometime later today and eventually will open an issue/pr in the upstream library.

**Reply by ganigeorgiev** [reply]
Most likely introduced with https://github.com/jacobmischka/svelte-flatpickr/commit/553b19301d168d3ad6ad814703707698cea90493 or https://github.com/jacobmischka/svelte-flatpickr/commit/96efff459ee8c673f09fcf471be2046d55a26f34

**Reply by ganigeorgiev** [reply]
I've submitted a "fix" in https://github.com/jacobmischka/svelte-flatpickr/pull/55.

_If the author doesn't manage to merge it by the end of the week, I'll apply a workaround and eventually release it as v0.12.2 release._
